src/components/TagsUI.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const FieldContainer = styled.div`
+  background: white;
+  width: 100%;
+  max-width: 600px;
+  min-height: 48px;
+  display: flex;
+  flex-wrap: wrap;
+  border: 1px solid #000;
+  border-radius: 4px;
+`;
+
+const TagTitle = styled.span`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 16px;
+  text-transform: capitalize;
+  line-height: 24px;
+  background-color: #008bf8;
+  padding: 6px 18px;
+  margin: 6px;
+  border-radius: 40px;
+`;
+
+const CrossIcon = styled.img`
+  height: 16px;
+  margin-left: 5px;
+  cursor: pointer;
+`;
+
+const InputWrapper = styled.div`
+  min-height: 48px;
+  min-width: 60px;
+  display: flex;
+  flex: 1;
+  overflow: hidden;
+`;
+
+const Form = styled.form`
+  display: flex;
+  flex: 1;
+`;
+
+const Input = styled.input`
+  flex: 1;
+  color: #008bf8;
+  border: transparent;
+  font-size: 16px;
+  line-height: 24px;
+  margin: 0;
+  padding: 12px;
+  outline: none;
+  font-family: "Inter", sans-serif;
+`;
+
+const Title = (props) => {
+  const { title, id } = props;
+  return (
+    <TagTitle>
+      <span>{title.text}</span>
+      <CrossIcon src={Cross} onClick={() => props.removeTitle(id)} />
+    </TagTitle>
+  );
+};
+
+const TagInput = (props) => {
+  let text, tagForm;
```

Comment:
I guess should use `useRef` here.

Reply:
How would you use?

Reply:
As you're not using `onSubmit` or something form related, I think you can simply use a `<div>`.